code/modules/supermatter/supermatter.dm

```diff
@@ -574,6 +607,30 @@ var/list/supermatter_final_thoughts = list(
 /obj/machinery/power/supermatter/get_artifact_scan_data()
 	return "Superdense crystalline structure - appears to have been shaped or hewn, lattice is approximately 20 times denser than should be possible."
 
+/obj/machinery/power/supermatter/proc/update_sound()
+	if(!sound_id)
+		sound_id = "[type]_[sequential_id(/obj/machinery/power/supermatter)]"
```

Comment:
Couldn't this just be `sound_id = "[sequential_id(type)]"`? 
Also any particular reason it isn't set in `Initialize()` rather than here?